### DISCUSSION TITLE
H4 on a Vevor MX-8x14

### DISCUSSION THREAD

**Discussion by duebelpapst** [kachurovskiy/nanoels#167] [Show and tell]
Hello everyone,

after I solved my problem with the Nanoels H4 / JMC closed loop / VFD, Maxim asked me for some photos of my wiring.
I thought it will be better to put the photos, and a description of what else I did on the lathe, in a new discussion. 
The solution, as described in the other [discussion](https://github.com/kachurovskiy/nanoels/discussions/151), was that I installed a ULN2803 in the step signal line.
Now for the lathe, what a piece of junk, I should have sent it back immediately.
The tool holder was made of aluminum, crazy Chinese, well I got a discount of 50€, and bought a multi-fix holder from Paulimot.
The bed was about 0.2 mm thinner at the chuck than at the other end, so I reground it as described on YouTube at [Aussished,](https://www.youtube.com/watch?v=bCtOT7TYrNU) about 9 to 10 hours, my arms almost fell off.
I also replaced the spindle bearings with tapered bearings and turned two new clamping nuts for them, the original ones were far too loose.
For the X-axis, I extended the lead screw to mount a timing belt pulley and screwed a bearing block to the carriage, to which I then attached the stepper, still with a 3D print, but hopefully soon made of aluminum.
For the Z-axis, I printed a flange for the JMC Nema 23, which will also be replaced by an aluminum flange.
I replaced the bad steel gib strips, on the cross slide and the top slide, with exactly matching ones made of brass, let's see if that was a good idea.
The best change was that I replaced the electronics and the motor with a 750 W three-phase motor with VFD. 
All in all, I'm happy with the lathe now, but there will definitely be a few more changes, e.g. a ball screw for the Z-axis, and I'll also rework the X-axis.
Next I have to make a cover for the timing belt and the drive train, and some covers for the ways and the energy chain of the X-axis stepper cable.

Kind regards,
Stefan 
![20240106_135750](https://github.com/kachurovskiy/nanoels/assets/45849605/86214d75-f45e-4d05-ae07-d66d418699e6)
![20240106_135729_HDR](https://github.com/kachurovskiy/nanoels/assets/45849605/1b4d25a8-c183-4f2d-9a57-8c5b94d161d4)
![20240106_135815](https://github.com/kachurovskiy/nanoels/assets/45849605/66ba505e-b1c7-4c89-a9ae-9d922a755ab1)
![20240106_135902](https://github.com/kachurovskiy/nanoels/assets/45849605/66c56cde-9419-40f4-8c2c-37d1f1bc849d)
![20240106_135910](https://github.com/kachurovskiy/nanoels/assets/45849605/6de8c533-9893-4b1c-a343-ca2dcbc405f1)
![20240106_135944](https://github.com/kachurovskiy/nanoels/assets/45849605/990d7f57-fca9-42da-9bf6-75770ca89394)
![20240106_135841](https://github.com/kachurovskiy/nanoels/assets/45849605/6b09b75b-e057-4ec3-a26f-391ca01a4e2f)
![20240106_135800](https://github.com/kachurovskiy/nanoels/assets/45849605/103cb342-62da-4145-85ab-7e273731220f)

**Comment by kachurovskiy** [comment]
Stefan, thanks a lot for sharing, very nice and original build 👍
